Source/SLLogger.h

```diff
@@ -0,0 +1,76 @@
+//
+//  SLLogger.h
+//  LoggingDemo
+//
+//  Created by 利辺羅 on 2014/12/09.
+//  Copyright (c) 2014年 SLF4C. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol SLLogger, SLMarker;
+
+typedef NS_ENUM(NSUInteger, SLLevel)
+{
+    SLLevelOff = 0,
+    SLLevelVerbose,
+    SLLevelDebug,
+    SLLevelInfo,
+    SLLevelWarning,
+    SLLevelError
+};
```

Comment:
This is pretty complex as the levels have to be hardcoded in the libraries log calls. If someone adds log levels they won't get used in shipping libraries, unless they get redirected to different levels, but then those would only be aliases to the existing levels.

I like the idea of extensibility, we should think more about it.